apps/docs/src/app/platform/component-docs/platform-forms/platform-time-picker/platform-time-picker-docs.component.html

```diff
@@ -0,0 +1,35 @@
+<fd-docs-section-title id="timePickerBasic" componentName="timePicker">
+    Basic Time Picker
+</fd-docs-section-title>
+<description>Basic usage of Time Picker without Forms and with options like <code>meredian</code>,
+    <code>allowNull</code>,
+    <code>disabled</code>, pre-populating and changing dates.
+</description>
+<component-example>
+    <fdp-time-picker-basic-example></fdp-time-picker-basic-example>
+</component-example>
+<code-example [exampleFiles]="timePickerBasic"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title id="timePickerReactive" componentName="timePicker">
```

Comment:
Use kebab-case for id attributes. `componentName` value needs to match component sub-page url (time-picker)

Reply:
@salarenko sorry, could you please clarify about `componentName` value? What do you mean?

I tried to find a pattern in the project for this, but there are a lot of camelCase and kebab-case in such cases, so
I will be glad on clarifying the rules for that.